### ISSUE TITLE
1007: ICE negotiation failed with 2.2

### ISSUE THREAD

**Issue by premar** [bigbluebutton/bigbluebutton#8808]
**Describe the bug**
In the new 2.2 version the FreeSWITCH config for the external ip is not working. I am using an server with an intern and external ip.

I set FreeSwitch up as mentioned in:
https://docs.bigbluebutton.org/2.2/configure-firewall.html#update-freeswitch

For setup of BBB i used the script and afterwards reinstalled the server manually. Both time with the same problem.

On my old 2.0 server I dont have this problem.

**Server config**
BigBlueButton Server 2.2.2 (1781)
                    Kernel version: 4.4.0-176-generic
                      Distribution: Ubuntu 16.04.6 LTS (64-bit)
                            Memory: 8174 MB
                         CPU cores: 8

/usr/share/bbb-web/WEB-INF/classes/bigbluebutton.properties (bbb-web)
       bigbluebutton.web.serverURL: https://bigbluebutton.domain.com
                defaultGuestPolicy: ALWAYS_ACCEPT
                 svgImagesRequired: true

/etc/nginx/sites-available/bigbluebutton (nginx)
                       server name: bigbluebutton.domain.com
                              port: 80, [::]:80
                              port: 443 ssl
                    bbb-client dir: /var/www/bigbluebutton

/var/www/bigbluebutton/client/conf/config.xml (bbb-client)
                Port test (tunnel): rtmp://bigbluebutton.domain.com
                              red5: bigbluebutton.domain.com
              useWebrtcIfAvailable: true

/opt/freeswitch/etc/freeswitch/vars.xml (FreeSWITCH)
                       local_ip_v4: 10.2xx.xxx.xxx
                   external_rtp_ip: 147.xxx.xxx.xxx
                   external_sip_ip: 147.xxx.xxx.xxx

/opt/freeswitch/etc/freeswitch/sip_profiles/external.xml (FreeSWITCH)
                        ext-rtp-ip: $${external_rtp_ip}
                        ext-sip-ip: $${external_sip_ip}
                        ws-binding: :5066
                       wss-binding: :7443

/usr/local/bigbluebutton/core/scripts/bigbluebutton.yml (record and playback)
                     playback_host: bigbluebutton.domain.com
                 playback_protocol: https
                            ffmpeg: 4.2.2-1bbb1~ubuntu16.04

/etc/bigbluebutton/nginx/sip.nginx (sip.nginx)
                        proxy_pass: 147.xxx.xxx.xxx

/usr/local/bigbluebutton/bbb-webrtc-sfu/config/default.yml (Kurento SFU)
                        kurento.ip: 10.2xx.xxx.xxx
                       kurento.url: ws://127.0.0.1:8888/kurento
                    localIpAddress: 10.2xx.xxx.xxx
               recordScreenSharing: true
                     recordWebcams: true
                  codec_video_main: VP8
               codec_video_content: VP8

/usr/share/meteor/bundle/programs/server/assets/app/config/settings.yml (HTML5 client)
                             build: 862
                        kurentoUrl: wss://bigbluebutton.domain.com/bbb-webrtc-sfu
                  enableListenOnly: true

**Expected behavior**
External user can connect.

**Actual behavior**
External user get "1007: ICE negotiation failed" message. Its understandable if the server is offering the internal ip.

**Screenshots**
Log from client:
![remote](https://user-images.githubusercontent.com/4980874/76992279-52276a80-694b-11ea-8378-8a6a58a7c531.JPG)

**Comment by capilkey** [comment]
The wss-binding value is incorrect. What you have:
```
/opt/freeswitch/etc/freeswitch/sip_profiles/external.xml (FreeSWITCH)
    ext-rtp-ip: $${external_rtp_ip}
    ext-sip-ip: $${external_sip_ip}
    ws-binding: :5066
    wss-binding: :7443
```
What it should be:
```
/opt/freeswitch/etc/freeswitch/sip_profiles/external.xml (FreeSWITCH)
    ext-rtp-ip: $${external_rtp_ip}
    ext-sip-ip: $${external_sip_ip}
    ws-binding: :5066
    wss-binding: 147.xxx.xxx.xxx:7443
```

**Comment by premar** [comment]
@capilkey thanks for the fast response.

After setting the wss-binding I got the 1002 errror. 

Additionally I had to set a dummy NIC as descripted in:
https://docs.bigbluebutton.org/2.2/configure-firewall.html#configure-a-dummy-nic-if-required

I assumed it worked in the first place, because I tested this command successfully:
curl --trace-ascii - -k https://EXTERNAL-IP-ADDRESS:443/bigbluebutton/api

Thanks for the help I'll close the issue.
